easybuild/easyblocks/t/tensorflow.py

```diff
@@ -894,6 +884,16 @@ def build_step(self):
 
         # Make TF find our modules. LD_LIBRARY_PATH gets automatically added by configure.py
         cpaths, libpaths = self.system_libs_info[1:]
+
+        # add absolute path to libnccl.so.2 directory provided by NCCL
+        # when LD_LIBRARY_PATH is filtered and LIBRARY_PATH is not
+        # filtered, e.g., in an environment such as EESSI
```

Comment:
Minor wording/wrapping enhancement:
```suggestion
        # when LD_LIBRARY_PATH is filtered and LIBRARY_PATH is not filtered.
        # E.g. in an environment such as EESSI.
```